assets/js/modules/tagmanager/components/settings/SettingsView.js

```diff
@@ -116,6 +131,24 @@ export default function SettingsView() {
 						</p>
 					</div>
 				) }
+
+				{ editViewSettingsURL && (
+					<div className="googlesitekit-settings-module__meta-item googlesitekit-settings-module__meta-item--data-only">
+						<p className="googlesitekit-settings-module__meta-item-data googlesitekit-settings-module__meta-item-data--tiny">
+							<Link href={ editViewSettingsURL } external>
+								{ createInterpolateElement(
+									__(
+										'Edit <VisuallyHidden>publication </VisuallyHidden>in Tag Manager',
```

Comment:
`<VisuallyHidden>publication </VisuallyHidden>` shouldn't be `publication`. If I'm not wrong, `publication` is related to Thank with Google module. It should be containers or something else. @aaemnnosttv, what should be within the `<VisuallyHidden>` component?

Reply:
Eagle eyes, thank you for spotting this, @hussain-t! This was a copy-paste mistake. I've changed it to `container` for now.